Task1_KFox/app/scss/styles.scss

```diff
@@ -19,49 +23,255 @@ $black: #000000;
+@mixin morelink($color, $pos, $posValue, $topValue) {
+    color: $color;
+    text-decoration: none;
+    font-size: 1rem; 
+    //position: relative;
+    padding-bottom: 2px;
+    //display: inline-block;
+                   
+    &::before {
+        content: '';
+        display: block;
+        width: 20px;
+        height: 20px;
+        border: 1px $yellow solid;
+        @include border-radius(10px);
+        text-align: center;
+        @if $topValue != 0 {
+            top: $topValue;
+        }
+        @else {
+            top: 0;
+        }
+        @if $pos == left {
+            left: $posValue; 
+        } @else if $pos == right {
+            right: $posValue;
+        }
+        position: absolute;        
+    }
+
+    &::after {
+        content: '';
+        display: block;
+        width: 7px;
+        height: 7px;
+        border-top: 1px $yellow solid;
+        border-right: 1px $yellow solid;            
+        @if $topValue != 0 {
+            top: $topValue+7px;
+        }
+        @else {
+            top: 7px;
+        }
+        @if $pos == left {
+            left: $posValue+5px; 
+        } @else if $pos == right {
+            right: $posValue+9px;
+        }
+        position: absolute;
+        transform: rotate(45deg);
+    }    
+} 
+ 
+@mixin border-radius($value) {
+    -webkit-border-radius: $value;
+    -moz-border-radius: $value;
+    -ms-border-radius: $value;
+    -o-border-radius: $value;
+    border-radius: $value;
+}
+
+* {
     margin: 0;
     padding: 0;
+    border: 0;
+}
+
+html {
     box-sizing: border-box;
     font-size: 100%;
 }
 
-body {
-    font-size: 62.5%;
+body {   
+    
+    box-sizing: border-box;    
+    //font-size: 62.5%;
+    font-weight: 300;
     background: $black;
     color: $white;
-    font-family: Tahoma, Verdana;
+    font-family: 'Source Sans Pro', 'Droid Sans', Tahoma, Verdana;
 }     
 
-div { 
+a { text-decoration: none }
+
+
+#landingPage {
+    height: 100vh;
+    //width: 100%;
+    overflow: hidden;
+    background-image: url(../images/nowplaying_bg.png);
+    background-size: cover;
+    padding: 0 1.3rem;
+    
+    header {
+        margin: 1.3rem 0;
+        position: relative;
+        text-align: center;
+        
+        #hamburger { 
+            position: absolute; 
+            left: 0;
+            top: 18px;
+            
+        }
+        #logo {}
+        #volume {
+            position: absolute;
+            right: 0;
+            top: 14px;
+        }
+    }
+    
+    #np-panel {        
+                
+        &-cover {
+            width: 100%;  
+            margin-bottom: 0.5rem;
+        }
+        
+        p { 
+            text-align: center;
+            font-size: 0.8rem;
+            
+            
+            &:nth-of-type(2) {
+                font-size: 1.4rem;
+            }
+        }
+        
+        &-controls {
+            margin: 1.3rem 0;
+            text-align: center;
+        }
+    }
+}
+
+
+section { 
```

Comment:
Be more specific, there could new sections appear in other places and your code will break them.